test/skimNtuple_HHbtag.cpp

```diff
@@ -1971,9 +1971,9 @@ int main (int argc, char** argv)
 		  bool metAccept       = passMETTrg    and trgRegions["met"]; 
 		  bool singletauAccept = passSingleTau and trgRegions["tau"];
 		  if (!isMC) {
-			legacyAccept    = legacyAccept    and !isMETDataset;
+			legacyAccept    = legacyAccept    and !isMETDataset and (!isTauDataset or pType!=2);
```

Comment:
doesn't this cause issues in TauTau? if pType==2, `LegacyAccept` is always false in the Legacy region

shouldn't it be `pType==2`, to basically bypass `!isTauDataset` in tauTau?

Reply:
Fixed, thanks.